packages/tools/devtools/devtools-browser-extension/src/devtools/DevtoolsScriptContent.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { formatDevtoolsScriptMessageForLogging } from "./Logging.js";
+
+/**
+ * Code run when "DevtoolsScript" executes.
+ * Factored out for testability.
+ */
+export function runDevtoolsScript(browser: typeof chrome): void {
```

Comment:
A bit late to the party 😄 but I take it this had to go into its own file so when importing it from the test we don't do the `import { browser } from "../Globals.js"; in the other file? Or could it have still lived there and imported by name from the test, but passed in the fake browser instance?